lib/daru/io/sql_data_source.rb

```diff
@@ -78,10 +97,19 @@ def init_adapter(db, query)
           DbiAdapter.new(db, query)
         when ActiveRecord::ConnectionAdapters::AbstractAdapter
           ActiveRecordConnectionAdapter.new(db, query)
+        when String
```

Comment:
I am not sure that `String` is enough condition to use SQLite adapter. If the API accepts strings at all, somebody could (rightfully) expect that `make_dataframe('mysql://user:password@database')` should also work.